Core/src/org/sleuthkit/autopsy/othercasessearch/OtherCasesSearchDialog.java

```diff
@@ -0,0 +1,369 @@
+    "OtherCasesSearchDialog.dialogTitle.text=Search Other Cases",
+    "OtherCasesSearchDialog.resultsTitle.text=Other Cases",
+    "OtherCasesSearchDialog.resultsDescription.text=Other Cases Search",
+    "OtherCasesSearchDialog.emptyNode.text=No results found.",
+    "OtherCasesSearchDialog.validation.invalidHash=The supplied value is not a valid MD5 hash."
+})
+/**
+ * The Search Other Cases dialog allows users to search for specific
+ * types of correlation properties in the Central Repository.
+ */
+final class OtherCasesSearchDialog extends javax.swing.JDialog {
+    private static final Logger logger = Logger.getLogger(OtherCasesSearchDialog.class.getName());
+    
+    private static final String FILES_CORRELATION_TYPE = "Files";
+    
+    private final List<CorrelationAttributeInstance.Type> correlationTypes;
+    private TextPrompt correlationValueTextFieldPrompt;
+
+    /**
+     * Creates a new instance of the Search Other Cases dialog.
+     */
+    OtherCasesSearchDialog() {
+        super((JFrame) WindowManager.getDefault().getMainWindow(), Bundle.OtherCasesSearchDialog_dialogTitle_text(), true);
+        this.correlationTypes = new ArrayList<>();
+        initComponents();
+        customizeComponents();
+    }
+    
+    /**
+     * Perform the other cases search.
+     */
+    private void search() {
+        new SwingWorker<List<CorrelationAttributeInstance>, Void>() {
+            
+            @Override
+            protected List<CorrelationAttributeInstance> doInBackground() {
+                List<CorrelationAttributeInstance.Type> correlationTypes;
+                List<CorrelationAttributeInstance> correlationInstances = new ArrayList<>();
+                
+                try {
+                    correlationTypes = EamDb.getInstance().getDefinedCorrelationTypes();
+                    for (CorrelationAttributeInstance.Type type : correlationTypes) {
+                        if (type.getDisplayName().equals((String) correlationTypeComboBox.getSelectedItem())) {
+                            correlationInstances = EamDb.getInstance().getArtifactInstancesByTypeValue(type, correlationValueTextField.getText());
+                            break;
+                        }
+                    }
+                } catch (EamDbException ex) {
+                    logger.log(Level.SEVERE, "Unable to connect to the Central Repository database.", ex);
+                } catch (CorrelationAttributeNormalizationException ex) {
+                    logger.log(Level.SEVERE, "Unable to retrieve data from the Central Repository.", ex);
+                }
+
+                return correlationInstances;
+            }
+
+            @Override
+            protected void done() {
+                try {
+                    super.done();
+                    List<CorrelationAttributeInstance> correlationInstances = this.get();
+                    DataResultViewerTable table = new CommonAttributesSearchResultsViewerTable();
```

Comment:
This table appears to be having some issues with resizing when it is created.  The following logger messages are appearing in the output.

SEVERE [org.sleuthkit.autopsy.commonfilesearch.CommonAttributesSearchResultsViewerTable]: Tried to set width on a column not supported by the CommonFilesSearchResultsViewerTable: Name
SEVERE [org.sleuthkit.autopsy.commonfilesearch.CommonAttributesSearchResultsViewerTable]: Tried to set width on a column not supported by the CommonFilesSearchResultsViewerTable: Known
SEVERE [org.sleuthkit.autopsy.commonfilesearch.CommonAttributesSearchResultsViewerTable]: Tried to set width on a column not supported by the CommonFilesSearchResultsViewerTable: Path
SEVERE [org.sleuthkit.autopsy.commonfilesearch.CommonAttributesSearchResultsViewerTable]: Tried to set width on a column not supported by the CommonFilesSearchResultsViewerTable: Comment
SEVERE [org.sleuthkit.autopsy.commonfilesearch.CommonAttributesSearchResultsViewerTable]: Tried to set width on a column not supported by the CommonFilesSearchResultsViewerTable: Device